src/templates/bootstrap5/file/form.ejs

```diff
@@ -67,8 +129,8 @@
 {% if (!ctx.disabled && (ctx.component.multiple || !ctx.files.length)) { %}
   {% if (ctx.self.useWebViewCamera) { %}
     <div class="fileSelector">
-      <button class="btn btn-primary" ref="galleryButton"><i class="{{ctx.iconClass('book')}}"></i> {{ctx.t('Gallery')}}</button>
-      <button class="btn btn-primary" ref="cameraButton"><i class="{{ctx.iconClass('camera')}}"></i> {{ctx.t('Camera')}}</button>
+      <button class="btn btn-primary" ref="galleryButton"><i class="fa fa-book"></i> {{ctx.t('Gallery')}}</button>
+      <button class="btn btn-primary" ref="cameraButton"><i class="fa fa-camera"></i> {{ctx.t('Camera')}}</button>
```

Comment:
Is there a reason we're hardcoding to font-awesome here? It looks like the previous function will (perhaps someday) allow us to opt in to another icon set, let's keep that around if it works please

Reply:
I'm sorry, just didn't notice that I updated it. Should be fixed now.